src/kibana/components/vislib/visualizations/_point_series_chart.js

```diff
@@ -100,22 +102,19 @@ define(function (require) {
 
       function callPlay(event) {
         var boundData = event.target.__data__;
+        var offsetX = $chartEl.offset().left;
         var wholeBucket = boundData && boundData.x != null;
 
+        // the min and max that the endzones start in
         var min = leftEndzone.w;
         var max = rightEndzone.x;
 
         // bounds of the cursor to consider
-        var xLeft = event.offsetX;
-        var xRight = event.offsetX;
-        if (wholeBucket) {
-          xLeft = xScale(boundData.x);
```

Comment:
and on master:

![2015-05-26 14_01_54](https://cloud.githubusercontent.com/assets/1329312/7823360/caf4cca2-03af-11e5-849e-86161bc2accb.gif)